apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerException.cs

```diff
@@ -93,6 +93,11 @@ public bool IsRetryable
         /// </summary>
         public TimeSpan? RecommendedRetryDelay { get; }
 
+        /// <summary>
+        /// Gets the underlying <see cref="Grpc.Core.RpcException"/>.
```

Comment:
I would reword "Gets" to "Returns", but more importantly point out that this may be null, if the SpannerException was not caused by an RpcException.